src/components/DataSourceSetup/index.module.less

```diff
@@ -0,0 +1,83 @@
+.head {
+  margin-bottom: 20px;
+}
+
+.iconWrapper {
+  display: flex;
+  align-items: center;
+  justify-content: center;
+
+  width: 48px;
+  height: 48px;
+
+  margin-right: var(--gap-base);
+  padding: 11px;
+
+  background: #fff;
+  border: 0.798px solid rgba(165, 165, 165, 0.3);
+  border-radius: 9.677px;
+  box-shadow: 0px 3.193568229675293px 12.774272918701172px 0px
+    rgba(8, 8, 8, 0.06);
+}
+
+.title {
+  margin: 0;
+
+  font-family: var(--font-manrope);
+}
+
+.text {
+  max-width: 858px;
+
+  font-family: var(--font-sora);
+}
+
+.label {
+  font-weight: 700;
+}
+
+.checkbox {
+  font-weight: 400;
+}
+
+.setupForm {
+  display: grid;
+  grid-template-columns: 1fr 1fr;
+  gap: var(--gap-base);
+
+  @media (max-width: 576px) {
+    grid-template-columns: 1fr;
+  }
+}
+
+.actions {
+  @media (max-width: 576px) {
+    display: flex;
+    flex-direction: column;
+  }
+}
+
+.back {
+  color: #470d69;
+
+  background: rgba(71, 13, 105, 0.1);
+  border: none;
+}
+
+.back,
+.submit {
+  width: 141px;
+
+  margin-right: var(--gap-5xs);
+
+  @media (max-width: 576px) {
+    width: 100%;
+    margin-right: 0;
+    margin-bottom: var(--gap-5xs);
+  }
```

Comment:
such media queries should be solved via antd grid system, it's possible. Using media queries is allowed in very rare cases when it's not possible to use the grid system